code/game/objects/items/devices/radio/headset.dm

```diff
@@ -113,6 +114,82 @@ GLOBAL_LIST_INIT(channel_tokens, list(
 	icon_state = "sec_headset"
 	worn_icon_state = "sec_headset"
 	keyslot = new /obj/item/encryptionkey/headset_sec
+	actions_types = list(/datum/action/item_action/dispatch)
+	var/radio_channel = RADIO_CHANNEL_SECURITY
+	COOLDOWN_DECLARE(dispatch_cooldown_timer)
+	var/dispatch_cooldown = 20 SECONDS
+
+/obj/item/radio/headset/headset_sec/ui_action_click(mob/user, action)
+	if(istype(action, /datum/action/item_action/dispatch))
+		dispatch(user)
+	else
+		to_chat("What else is there, moron?")
```

Comment:
use CRASH instead of a to_chat that will runtime due to improper args

Reply:
ab00258